homeworks/dobrikbartaeva/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,12 @@
+name = (input("Enter your name: ")).upper().strip() 
+#mit jelent a nagy betűvel? az egész nagybetű? Én annak vettem... Vagy nagy kezdőbetű? mert akkor .capitalize() a .upper() helyett
+
+age_years = int(input("Enter your age (in years): "))
+age_days = age_years * 365
+py_experience_years = int(input("Enter your Python experience in years: "))
+
+prof_ambitions = (input("Would you like to be a professional? (yes/no): ")).lower().strip()
+prof_ambitions_text = "He/she wants to be a Python developer!" if prof_ambitions == "yes" else "He/she does not want to be a Python developer!"
```

Comment:
Nagyon szuper a ternary operator.
Egy fejlesztési lehetőség: elég lenne a "wants" és a "does not want", a többi rész mehet a print-be, így nincs ismétlés.

Reply:
makes sense - átírtam